Tools/clinic/clinic.py

```diff
@@ -221,6 +221,31 @@ def c_repr(s: str) -> str:
     return '"' + s + '"'
 
 
+def wrapped_c_string_literal(line, width=72):
+    add, out = text_accumulator()
+    words = line.split(' ')
+    add(' \\\n')
+    while True:
+        add('"')
+        sz = 1
+        while True:
+            try:
+                word = words[0]
+            except IndexError:
+                add('"')
+                return out()
+            sz += len(word) + 1
+            if sz > 72:
+                break
+            add(word)
+            del words[0]
+            if words:
+                add(' ')
+        add('"')
+        add(' \\')
+        add('\n')
```

Comment:
Why not a single `add()`?

Reply:
I adjusted the code to have custom suffix; line continuation is only needed for the preprocessor. So this code is now slightly different.